library/src/test/java/com/datatorrent/lib/formatter/JsonFormatterTest.java

```diff
@@ -198,15 +199,25 @@ public void testJSONToPOJONoFieldPOJO()
   @Test
   public void testApplication() throws IOException, Exception
   {
-    try {
-      LocalMode lma = LocalMode.newInstance();
-      Configuration conf = new Configuration(false);
-      lma.prepareDAG(new JsonFormatterApplication(), conf);
-      LocalMode.Controller lc = lma.getController();
-      lc.run(2000);// runs for 2 seconds and quits
-    } catch (ConstraintViolationException e) {
-      Assert.fail("constraint violations: " + e.getConstraintViolations());
-    }
+    LocalMode lma = LocalMode.newInstance();
+    Configuration conf = new Configuration(false);
```

Comment:
@tweise Purpose was to check for serialization of the DAG  and to check if app runs with the operator under test. One approach was to just create and run a sample application for some time. Assertions were not included in that application test as other unit tests had covered those. But the downside was that it created a lot of console outputs.
Hence this approach was followed.

In general, do we need to add a test to verify serialization of the operator ? If yes, could you advise on correct approach ?  I will modify it accordingly.

@yogidevendra Thank you for providing the examples. However, unit tests already covers it.